tests/extensions/wcf/Microsoft.DotNet.UpgradeAssistant.Extensions.WCFUpdater.Test/WCFUpdateStepTest.cs

```diff
@@ -42,6 +43,16 @@ public WCFUpdateStepTest(ITestOutputHelper output)
         [InlineData(Proj, "TestInputFiles\\MultiServicesSourceCode.txt", "TestInputFiles\\MultiServicesConfig.txt", "", UpgradeStepStatus.Incomplete)] // Multiple services case
         public void WCFUpdateTest(string proj, string main, string config, string directive, UpgradeStepStatus expected)
         {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                return;
+            }
+
+            proj = proj.Replace('\\', Path.DirectorySeparatorChar);
+            main = main.Replace('\\', Path.DirectorySeparatorChar);
+            config = config.Replace('\\', Path.DirectorySeparatorChar);
+            directive = directive.Replace('\\', Path.DirectorySeparatorChar);
+
```

Comment:
Do we still need this if this only runs on Windows anyway?

Reply:
Technically? Probably not, but it doesn't hurt.